training/Train.py

```diff
@@ -80,6 +70,12 @@ def main():
             Utils.csvwrite('logs/trainloss.csv',\
                            [epoch,epoch_train_loss.average()])
 
```

Comment:
I've empirically found the current PR to be stable. I'm not sure if those changes could threaten the stability.